src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements Drawable, AreaCalculable {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. It's better to separate concerns, so consider creating different interfaces for drawing and calculating the area.